index.html

```diff
@@ -34,5 +34,131 @@
+          </div>
+        </div>
+            <button class="buttons"type="button">See Project</button>
+        </div>
+        <div class="pic_div">
+          <div class="pro_div">
+          <div class="cool">Professional Art printing Data</div>
+          <div class="yes">A daily selection of privately personalised reads, no accounts or sign-ups required and has been the industry standard.</div>
+          <div class="fine">
+            <div class="html">html</div>
+            <div class="boot">bootstrap</div>
+            <div class="ruby">Ruby</div>
+          </div>
+        </div>
+          <button class="buttons"type="button">See Project</button>
+        </div>
+        <div class="pic_div">
+        <div class="pro_div">
+          <div class="cool">Professional Art printing Data</div>
+          <div class="yes">A daily selection of privately personalised reads, no accounts or sign-ups required and has been the industry standard.</div>
+          <div class="fine">
+            <div class="html">html</div>
+            <div class="boot">bootstrap</div>
+            <div class="ruby">Ruby</div>
+          </div>
+          </div>
+          <button class="buttons"type="button">See Project</button>
+        </div>
+        <div class="pic_div">
+        <div class="pro_div">
+          <div class="cool">Professional Art printing Data</div>
+          <div class="yes">A daily selection of privately personalised reads, no accounts or sign-ups required and has been the industry standard.</div>
+          <div class="fine">
+            <div class="html">html</div>
+            <div class="boot">bootstrap</div>
+            <div class="ruby">Ruby</div>
+          </div>
+          </div>
+          <button class="buttons"type="button">See Project</button>
+        </div>
+      <div class="pic_div">
+        <div class="pro_div">
+          <div class="cool">Professional Art printing Data</div>
+          <div class="yes">A daily selection of privately personalised reads, no accounts or sign-ups required and has been the industry standard.</div>
+          <div class="fine">
+            <div class="html">html</div>
+            <div class="boot">bootstrap</div>
+            <div class="ruby">Ruby</div>
+          </div>
+        </div>
+         <button class="buttons"type="button">See Project</button>
+        </div>
+      <div class="pic_div">
+        <div class="pro_div">
+          <div class="cool">Professional Art printing Data</div>
+          <div class="yes">A daily selection of privately personalised reads, no accounts or sign-ups required and has been the industry standard.</div>
+          <div class="fine">
+            <div class="html">html</div>
+            <div class="boot">bootstrap</div>
+            <div class="ruby">Ruby</div>
+          </div>
+         </div>    
+       <button class="buttons"type="button">See Project</button>
+         </div>
+         </div>
+     <div id="about_section">
```

Comment:
Please kindly add the appropriate font size and font-color in yout about section your font is blue please add the correct font color which can be found by inspection the figma template.l